test/ParsedRequestBuilderTest.cpp

```diff
@@ -125,28 +125,57 @@ TEST(ParsedRequestBuilderTest, extractOperationIfSpecified) {
 }
 
 // _____________________________________________________________________________________________
-TEST(ParsedRequestBuilderTest, isGraphStoreOperation) {
-  auto isGraphStoreOperation =
+TEST(ParsedRequestBuilderTest, isGraphStoreOperationIndirect) {
+  auto isGraphStoreOperationIndirect =
       CPP_template_lambda()(typename RequestT)(const RequestT& request)(
           requires ad_utility::httpUtils::HttpRequest<RequestT>) {
     const auto builder = ParsedRequestBuilder(request);
-    return builder.isGraphStoreOperation();
+    return builder.isGraphStoreOperationIndirect();
   };
-  EXPECT_THAT(isGraphStoreOperation(makeGetRequest("/")), testing::IsFalse());
-  EXPECT_THAT(
-      isGraphStoreOperation(makeGetRequest("/?query=foo&access-token=bar")),
-      testing::IsFalse());
-  EXPECT_THAT(isGraphStoreOperation(makeGetRequest("/?default")),
+  EXPECT_THAT(isGraphStoreOperationIndirect(makeGetRequest("/")),
+              testing::IsFalse());
+  EXPECT_THAT(isGraphStoreOperationIndirect(
+                  makeGetRequest("/?query=foo&access-token=bar")),
+              testing::IsFalse());
+  EXPECT_THAT(isGraphStoreOperationIndirect(makeGetRequest("/?default")),
               testing::IsTrue());
-  EXPECT_THAT(isGraphStoreOperation(makeGetRequest("/?graph=foo")),
+  EXPECT_THAT(isGraphStoreOperationIndirect(makeGetRequest("/?graph=foo")),
               testing::IsTrue());
-  EXPECT_THAT(isGraphStoreOperation(
+  EXPECT_THAT(isGraphStoreOperationIndirect(
                   makeGetRequest("/default?query=foo&access-token=bar")),
               testing::IsFalse());
 }
 
 // _____________________________________________________________________________________________
-TEST(ParsedRequestBuilderTest, extractGraphStoreOperation) {
+TEST(ParsedRequestBuilderTest, isGraphStoreOperationDirect) {
+  auto isGraphStoreOperationDirect =
+      CPP_template_lambda()(typename RequestT)(const RequestT& request)(
+          requires ad_utility::httpUtils::HttpRequest<RequestT>) {
+    const auto builder = ParsedRequestBuilder(request);
+    return builder.isGraphStoreOperationDirect();
+  };
+  EXPECT_THAT(isGraphStoreOperationDirect(makeGetRequest("/")),
+              testing::IsFalse());
+  EXPECT_THAT(isGraphStoreOperationDirect(
+                  makeGetRequest("/?query=foo&access-token=bar")),
+              testing::IsFalse());
+  EXPECT_THAT(isGraphStoreOperationDirect(makeGetRequest("/http-graph-store")),
+              testing::IsTrue());
+  EXPECT_THAT(
+      isGraphStoreOperationDirect(makeGetRequest("/http-graph-store/foo")),
+      testing::IsTrue());
+  EXPECT_THAT(
+      isGraphStoreOperationDirect(makeGetRequest("/http-graph-store/foo?bar")),
+      testing::IsTrue());
+  EXPECT_THAT(
+      isGraphStoreOperationDirect(makeGetRequest("/foo/http-graph-store")),
+      testing::IsFalse());
+  EXPECT_THAT(isGraphStoreOperationDirect(makeGetRequest(":")),
+              testing::IsFalse());
+}
```

Comment:
1. Write a lambda that abstracts away `isGraphOpDirec(makeGetRequest)`, 
2. Use `EXPECT_TRUE` and `EXPECT_FALSE` (much more concise).